docs/src/components/Layout/Header.tsx

```diff
@@ -20,19 +20,19 @@ import { Logo } from '@/components/Logo';
 import { FrameworkChooser } from './FrameworkChooser';
 
 const NavLink = ({ href, children, isExternal = false }) => {
-  const router = useRouter();
-  const isCurrent = router.pathname.startsWith(href);
+  const { pathname, query } = useRouter();
+  const isCurrent = pathname.startsWith(href);
```

Comment:
Could we have a check here to make sure `href` isn't just "/" so the logo doesn't have the current styling for every page since every pathname starts with "/"? Or maybe there is a better way?

Reply:
OK! Yes, I'll add that.